src/nxdoc/nuxeo-server/administration/database-configuration/mariadb.md

```diff
@@ -0,0 +1,41 @@
+---
+title: MariaDB
+review:
+    comment: ''
+    date: '2017-07-12'
+    status: ok
+labels:
+    - lts2016-ok
+    - database
+    - mariadb
+    - multiexcerpt-include
+toc: true
+tree_item_index: 500
+
+---
+The Nuxeo Platform supports the following MariaDB version:
+
+{{{multiexcerpt 'MariaDB-supported' page='Compatibility Matrix'}}}
+
+## Configuration
+
+For Nuxeo versions up to and including **LTS 2016**, configuration of MariaDB server is exactly the same as for [MySQL]({{page page='mysql'}}), using the **MySQL driver** and `mysql` template.
```

Comment:
Suggested change:

    using the `mysql` template, which includes the standard **MySQL JDBC driver**.